web-app/src/components/UndoCard.tsx

```
@@ -0,0 +1,193 @@
+  UndoneDate: string | null;
+  onUndo: (id: number) => void;
+  paperlessUrl: string;
+}
+
+const formatDate = (dateString: string | null): string => {
+  if (!dateString) return '';
+
+  try {
+    const date = new Date(dateString);
+    // Check if date is valid
+    if (isNaN(date.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)